build.gradle

```
@@ -19,12 +19,23 @@ repositories {
 }
 
 dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+
 	compileOnly 'org.projectlombok:lombok'
 	developmentOnly 'org.springframework.boot:spring-boot-devtools'
 	annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
 	annotationProcessor 'org.projectlombok:lombok'
+
+	compile group: 'org.hibernate', name: 'hibernate-validator', version: '6.0.10.Final'
```

이렇게 바꿀 수 있을 것 같은데 이렇게 명시해주신 이유가 있으실까요?

```suggestion
	implementation 'org.hibernate:hibernate-validator:6.0.10.Final'
```

---

수정하도록 하겠습니다.!